lib/matplotlib/patches.py

```diff
@@ -127,6 +127,29 @@ def contains(self, mouseevent, radius=None):
         """
         Test whether the mouse event occurred in the patch.
 
+        Parameters
+        ----------
+        mouseevent : .MouseEvent
+            Where the user clicked.
+
+        radius : float, optional
+            Additional margin on the patch in target coordinates of
+            ``self.get_transform()``. See `.Path.contains_point` for further
+            details.
+
+            If `None` the default value depends on state of the object
+
+            - If `matplotlib.Artist.get_picker` is a number, defaults
+              to that value.  This is so that picking works as expected.
```

Comment:
```suggestion
            - If `matplotlib.Artist.get_picker` is a number, the default
              is that value.  This is so that picking works as expected.
```